src/clients/Device.js

```diff
@@ -909,41 +870,39 @@ class Device extends EventEmitter {
   /**
    * Checks our cache to see if we have the function state, otherwise requests
    * it from the core, listens for it, and resolves our deferred on success
-   * @returns {*}
    */
   _ensureWeHaveIntrospectionData = async (): Promise<*> => {
     if (this._hasFunctionState()) {
-      return Promise.resolve();
+      return;
     }
 
     try {
       this.sendMessage('Describe');
-      const token = this.sendMessage('Describe');
+      this.sendMessage('Describe');
```

Comment:
At least that's what it looks like... Maybe I made a mistake in the code and call it twice... :/